src/services/usercentrics-service.ts

```diff
@@ -27,36 +30,61 @@ const subscribe = (x: (x: ServiceState) => void): (() => void) => {
     }
 }
 
-const setIsOpen = (isOpen: boolean) => {
+const setIsOpen = (isOpen: boolean, hasInteracted?: boolean) => {
```

Comment:
... maybe we could set it to true in any case?

Reply:
I don't think it's technically correct to set `hasInteracted: true` when the Usercentrics dialog opens, because it's possible user simply doesn't click anything. Maybe they adblock it so that the code runs, but it has `display: none;`. 🤔